3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentManager.scala

```diff
@@ -270,6 +292,21 @@ object EnrichmentManager {
       case None => ip
     })
 
+    val tuple = (trCurrency,TransactionAmounts(trTotal, trTax, trShipping), tiPrice, tiCurrency)
+
+    val currencyConversion = {
+      registry.getCurrencyConversionEnrichment match {
+        case Some(currency) => {
+          Option(tuple) match {
+            case Some(cuP) =>
+              var currencyConversion = currency.convertCurrencies(cuP._1, cuP._2, cuP._3, cuP._4)
+            case None => unitSuccess // No fields updated
```

Comment:
How is Option((trCurrency,TransactionAmounts(trTotal, trTax, trShipping), tiPrice, tiCurrency)) ever going to be None? It's never going to be None.